pebblo/app/config/config.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+
+from pydantic import BaseSettings, Field
+import pathlib
+
+# Default config value
+DEFAULT_SERVICE_CONFIG = './config_default.yaml'
+dir_path = pathlib.Path().absolute()
+
+
+# Port BaseModel
+class PortConfig(BaseSettings):
+    host: str = Field(default='localhost')
+    port: int = Field(default=37000)
+
+
+# Report BaseModel
+class ReportConfig(BaseSettings):
+    format: str = Field(default='pdf')
+    outputDir: str = Field(dir_path)
+
+
+# Logging BaseModel
+class LoggingConfig(BaseSettings):
+    level: str = Field(default='info')
+
+
+# ConfigFile BaseModel
+class Config(BaseSettings):
+    daemon:  PortConfig
+    reports: ReportConfig
+    logging: LoggingConfig
+
+
+def load_config(path) -> Config:
+    try:
+        # If Path does not exist in command, set default config value
+        conf_obj = Config(
+            daemon=PortConfig(
+                host='0.0.0.0',
+                port=3700
+            ),
+            reports=ReportConfig(
+                format='pdf',
+                outputDir='./config.py'
```

Comment:
Why otuputDir is a file name?

Reply:
Updated outputdir 
resolving conversation.